kolibri/core/logger/api.py

```diff
@@ -11,6 +11,7 @@
 from django.db.models import Subquery
 from django.db.models import Sum
 from django.db.models import Value
+from django.db.models.functions import Coalesce
```

Comment:
today I learned...

Reply:
I think it was Sir Mixalot who said: If you don't want `None` then coalesce your sum!